src/internal/observable/FromEventObservable.ts

```diff
@@ -206,7 +206,7 @@ export class FromEventObservable<T> extends Observable<T> {
     } else if (isEventTarget(sourceObj)) {
       const source = sourceObj;
       sourceObj.addEventListener(eventName, <EventListener>handler, <boolean>options);
-      unsubscribe = () => source.removeEventListener(eventName, <EventListener>handler);
+      unsubscribe = () => source.removeEventListener(eventName, <EventListener>handler, <boolean>options);
```

Comment:
The `options` aren't necessarily a `boolean`, they can be an object as well.

Reply:
I'm unaware of why the `boolean` assertion was used in the original code's call to the `addEventListener` method. Perhaps the typings in an earlier version of TypeScript were not representative of the API? The [current typings](https://github.com/Microsoft/TypeScript/blob/v2.7.2/lib/lib.dom.d.ts#L14638-L14645) seem okay, so I guess this could be changed. However, it is only a type assertion and has no runtime effect.